### PR TITLE
75 card components usage

### PR DESCRIPTION
**Pull request by TowerOneka**

### CODE REVIEW

**Comment by ukorvl** [comment]
Consider to fix previous 2 pull requests, which are included here, i don't see other issues.

**Review by ukorvl — APPROVED** [review]

File: site/src/components/common/PostCard/PostCard.tsx [hunk 1]
```diff
@@ -1,13 +1,9 @@
...
 import cx from 'classnames'
-import { string, shape, bool, InferProps } from 'prop-types'
-
-import Icon from 'components/Icon'
-import Button from 'components/Button'
 
 import s from './PostCard.module.scss'
-import { getBlogPosts } from 'src/strapi'
 import { MappedBlog } from 'src/strapi/types/entities'
+import Card from 'components/Card'
```
Comment (ukorvl) [inline site/src/components/common/PostCard/PostCard.tsx:6]: Now Card is not a default export

Comment (ukorvl) [reply site/src/components/common/PostCard/PostCard.tsx:6]: https://github.com/NilFoundation/nil.foundation/actions/runs/6704650284/job/18217466130?pr=84

Comment (TowerOneka) [reply site/src/components/common/PostCard/PostCard.tsx]: Update that

**Pull request merged**